metadata-models/src/main/pegasus/com/linkedin/secret/DataHubSecretValue.pdl

```diff
@@ -24,4 +26,15 @@ record DataHubSecretValue {
    * Description of the secret
    */
   description: optional string
+
```

Comment:
Should we add an updated just to keep this future proof?

Reply:
We will update once we have that! Thanks